test/test_core.py

```diff
@@ -6564,23 +6561,18 @@ def test_relocatable_void_function(self):
     self.do_core_test('test_relocatable_void_function.c')
 
   @wasm_simd
-  def test_wasm_intrinsics_simd(self):
-    def run():
-      self.do_runf('test_wasm_intrinsics_simd.c', 'Success!')
+  @parameterized({
+    '': ([],),
+    'unsigned_char': (['-funsigned-char'],),
+  })
+  def test_wasm_intrinsics_simd(self, args):
     # Improves test readability
```

Comment:
This comment goes with the `-Wno-c++11-narrowing`, but it looks like that flag was mistakenly dropped.

Reply:
But this test is not doesn't contain any such warnings.. and its not even a C++ test.

I.e. if I can remove the `-Wno-xxx` flag and no warnings show up then its not doing anything right?

Reply:
Oh interesting, I guess so. We should just remove the comment, too, then.